### ISSUE TITLE
Is dwelving delving?

### ISSUE THREAD

**Issue by eccstartup** [adambard/learnxinyminutes-docs#1156]
[Here](https://github.com/adambard/learnxinyminutes-docs/blob/master/perl6.html.markdown)

```
## Meta operators !
# Oh boy, get ready. Get ready, because we're dwelving deep
#  into the rabbit's hole, and you probably won't want to go
#  back to other languages after reading that.
#  (I'm guessing you don't want to already at that point).
# Meta-operators, as their name suggests, are *composed* operators.
# Basically, they're operators that apply another operator.
```

**Comment by eccstartup** [comment]
@vendethiel again

**Comment by vendethiel** [comment]
yes, seen that. good catch :)
